src/Commands/Env/DeployCommand.php

```diff
@@ -15,29 +15,34 @@ class DeployCommand extends TerminusCommand implements SiteAwareInterface
     use SiteAwareTrait;
 
     /**
-     * Deploy the dev environment to either test or live
+     * Deploys code to the Test or Live environment.
```

Comment:
I guess it's a question of whether these are the command-ified names, or whether this is their proper name. I named this as proper name with capitalization because of the Docs style guide: https://github.com/pantheon-systems/documentation/blob/master/style-guide.md#environments